parsons/__init__.py

```diff
@@ -73,6 +73,7 @@
     ("parsons.mobilize_america.ma", "MobilizeAmerica"),
     ("parsons.nation_builder.nation_builder", "NationBuilder"),
     ("parsons.newmode.newmode", "Newmode"),
+    ("parsons.newmode.newmode_v2", "NewmodeV2"),
```

Comment:
See my comment in the main thread for thoughts on overall design of the deprecation process